src/components/vars.sass

```diff
@@ -69,3 +69,11 @@ $top-bar-width: 960px
 // bottom bar
 //
 $bottom-bar-height: 300px
+
+.no-jitter
```

Comment:
I don't suppose there was a way for us to apply this more globally, is there?

Reply:
I didn't have any great ideas.